include/calculate.h

```diff
@@ -106,8 +106,9 @@ struct _calculate_c_library {
 #ifdef __cplusplus
 }
 #else
-extern const struct _calculate_c_library _c_calculate;
-const struct _calculate_c_library* const calculate = &_c_calculate;
+#define calculate_init() \
+const struct _calculate_c_library* const calculate = _get_calculate_c_library()
```

Comment:
The use of a function returning a pointer instead of an object workarounds the necessity of prepending `__declspec(dllimport)` to a global object under  **Windows**. In addition, the `calculate_init` macro is meant to be used inside main as static global pointers cannot be set in **C**.